content/topics/coding-horrors/tdd-horrors/_index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: TDD Horrors
```

Comment:
FOR REAL SERIOUS QUESTION THAT I ACTUALLY WANT ANSWERED:

Why not just put all this stuff with the rest of the unit testing topic stuff? Why do we want to scatter this content around? There is a testing topic already

We are coders. We like cohesion. Why do you not like cohesion? 

I've asked this before. More than once. Please answer now.

Reply:
Since TDD and Unit testing aren't the same thing that got me all confused